libs/features/webapp-prompt/webapp-prompt.js

```diff
@@ -35,6 +39,33 @@ const getIcon = (content) => {
   return icons.company;
 };
 
+const showTooltip = (element, message, time = 5000) => {
+  element.setAttribute('data-pep-dismissal-tooltip', message);
+  setTimeout(() => {
+    element.removeAttribute('data-pep-dismissal-tooltip');
+  }, time);
+};
+
+const playFocusAnimation = (element, iterationCount = 2, animationDuration = 2500) => {
+  document.documentElement.style.setProperty('--animation-duration', `${animationDuration}ms`);
```

Comment:
As suggested in the CSS file, I'd avoid defining this globally. Could the value not be attached to the element itself?

Reply:
I've addressed this as part of the earlier comment